src/request.ts

```diff
@@ -29,6 +30,7 @@ export async function request<Query, Variables>(
     const headers = {
       Authorization: `Bearer ${ctx.apiKey}`,
       'Content-Type': 'application/json',
+      'X-Plain-SDK-Version': packageJson.version,
```

Comment:
Is it worth adding 'typescript-sdk' somewhere in the value or name so we can support other sdks in future?